src/scvmm/.vscode.example/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Do you need to submit `.vscode.example` folder?

Reply:
`.vscode.example` and `setup.md` are convenient for the developers. It is not required for the end user. Should I delete these files?